workflow-templates/im-test-cypress-schedule.yml

```diff
@@ -0,0 +1,203 @@
+# Workflow Code: SwimmingBear_v1    DO NOT REMOVE
+# Purpose:
+#    Runs Cypress UI Health Check tests in Github Actions on a schedule.
+#    With the workflow you specify when to run this test.
+#
+# Frequency:
+#    - This workflow is intended to be reusable, and used once per repository.
```

Comment:
Why only once per repository? If there are multiple cypress test projects that need to be run, could they not be handled by different workflows? The npm script could run them all, but I don't know how the results would look afterwards. Maybe it would be fine. Either way, is there a strong reason for there to only be one of these workflows per repository?

Reply:
I have updated the frequency as `This workflow can be duplicated once per cypress project that you wish to run independently.` This was intended mainly for UI healthcheck tests which should have a single project under a repository but yes we can reuse this workflow for other Cypress projects as well. Thanks for the feedback.